Sources/HaCWebsiteLib/Utils/DatabaseUtils.swift

```diff
@@ -28,34 +28,36 @@ extension URL {
   }
 }
 
-func testDatabase() {
-  do {
-    guard let databaseURLString = DotEnv.get("DATABASE_URL"),
-      let databaseURL = URL(string: databaseURLString),
-      let databaseURLComponents = databaseURL.databaseURLComponents else {
-      fatalError("Couldn't find valid DATABASE_URL environment variable")
-    }
+func getDatabaseDriver() throws -> PostgreSQLDriver.Driver {
+  guard let databaseURLString = DotEnv.get("DATABASE_URL"),
+    let databaseURL = URL(string: databaseURLString),
+    let databaseURLComponents = databaseURL.databaseURLComponents 
+  else {
+    fatalError("Couldn't find valid DATABASE_URL environment variable")
+  }
+  let driver = try Driver(
+    masterHostname: databaseURLComponents.host,
+    readReplicaHostnames: [],
+    user: databaseURLComponents.user,
+    password: databaseURLComponents.password,
+    database: databaseURLComponents.database
+  )
+  return driver
+}
 
-    let driver = try Driver(
-      masterHostname: databaseURLComponents.host,
-      readReplicaHostnames: [],
-      user: databaseURLComponents.user,
-      password: databaseURLComponents.password,
-      database: databaseURLComponents.database
-    )
+func prepareDatabase() {
```

Comment:
We're trying to avoid public/internal top level functions (correct @jaredkhan?) as it's not really idiomatic Swift (fine in the scope of a single file).

Could we namespace this in an enum?

Reply:
Yeah whacking it in a DatabaseUtils enum could be nice